docs/sitegen.yaml

```diff
@@ -250,7 +250,14 @@ backend:
                     - "mp/cors/hide*.adoc"
                   includes:
                     - "mp/cors/*.adoc"
-
+            - title: "Fault Tolerance"
+              pathprefix: "/mp/faulttolerance"
+              glyph:
+                type: "icon"
+                value: "share"
+              items:
+                  includes:
```

Comment:
I could not build the site locally without adding the `-` list-item indicator in front of `includes` on this line.